go.mod

```diff
@@ -6,16 +6,14 @@ require (
 	// TODO: update version once it is officially released
 	github.com/container-storage-interface/spec v1.10.1-0.20241022120259-f6b6d53db606
 	github.com/evanphx/json-patch v5.9.0+incompatible
-	github.com/fsnotify/fsnotify v1.8.0
 	github.com/golang/mock v1.6.0
-	github.com/google/gofuzz v1.2.0
+	github.com/google/gofuzz v1.2.0 // indirect
```

Comment:
This indirect dependency should appear in the second section, where indirect dependencies are.

Reply:
yeah I did that, seems on doing go.mod, it got replaced again, will update it